.circleci/config.yml

```diff
@@ -35,7 +38,7 @@ jobs:
               | tar -xvz --strip=1 -C /usr/local/kubebuilder
       - run:
           name: Run tests
-          command: go test -race -count=1 -v ./...
+          command: ginkgo -race -r
```

Comment:
Can we add the `-v` back in? I reckon it's always useful to have the extra info, because you're only going to look at the output if the tests fail

Reply:
Can do! For the future, passing tests can be made silent and failing ones noisy by writing logs and other output to `ginkgo.GinkgoWriter`, which is an io.Writer that bins its buffer if the test passes, and spews it to stdout if it fails.